examples/with-emotion-fiber/shared/emotionCache.js

```diff
@@ -0,0 +1,2 @@
+import createCache from '@emotion/cache'
+export const emotionCache = createCache()
```

Comment:
this works, but if you dont need to configure cache you might want to just reuse the one in `emotion` package
```js
import { cache } from 'emotion'
``

Then you could also just use `emotion-server` instead of `create-emotion-server` (for simplicity).